crates/router/src/connector/bambora/transformers.rs

```diff
@@ -203,6 +300,29 @@ pub struct BamboraPaymentsResponse {
     risk_score: Option<f32>,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct Bambora3DsResponse {
+    #[serde(rename = "3d_session_data")]
+    three_d_session_data: String,
+    contents: String,
+}
+
+#[derive(Debug, Serialize, Default, Deserialize)]
+pub struct BamboraMeta {
+    pub three_d_session_data: String,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct BamboraThreedsContinueRequest {
+    pub(crate) payment_method: String,
+    pub card_response: CardResponse,
+}
+
+#[derive(Default, Debug, Deserialize, Serialize, Eq, PartialEq)]
```

Comment:
Is Serialize required for this struct?

For response struct's Serialize not required.